### PR TITLE
Adds Gun Cases. For guns.

### PR DESCRIPTION
**Pull request by Martinpachu**
## About The Pull Request

Adds in gun cases that can hold a variety of guns and magazines, intended for use in cargo and certain ships and ruins. Sprites by retlaw are a work in progress.

## Why It's Good For The Game

Adds in a cool way to store firearms safely, and also its realistic i think. Also requested by apogee.

## Changelog

:cl:
add: Cargo guns and certain guns in ships now come inside cases alongside some magazines, the ammo now comes separately for safety reasons.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic.dm [hunk 1]
```diff
@@ -96,15 +96,16 @@
 
 /obj/item/gun/ballistic/Initialize()
 	. = ..()
-	if (!spawnwithmagazine)
+	if (!spawnwithmagazine && !istype(magazine, /obj/item/ammo_box/magazine/internal))
```
Comment (SomeguyManperson) [inline code/modules/projectiles/guns/ballistic.dm]:
```suggestion
	if (!spawnwithmagazine && !ispath(mag_type, /obj/item/ammo_box/magazine/internal))
```
I goofed here

**Comment by Martinpachu** [comment]
Should be **ready.** A **test-merge** would be good.

**Comment by rye-rice** [comment]
fix your merge conflict!!

File: code/modules/cargo/packs/gun.dm [hunk 2]
```diff
@@ -9,53 +9,52 @@
 /datum/supply_pack/gun/disposable
 	name = "Disposable Gun Crate"
 	desc = "In some sectors, these disposable pistols are the only firearms that can be legally sold for less than 200cr. That price is still far too high."
-	cost = 150
-	contains = list(/obj/item/gun/ballistic/automatic/pistol/disposable)
+	cost = 300
```
Comment (FalloutFalcon) [inline code/modules/cargo/packs/gun.dm:12]: "sold for less then 200cr." 300cr

**Review by FalloutFalcon — DISMISSED** [review]

**Pull request merged**